cmd/sentry/options/options.go

```diff
@@ -81,6 +83,7 @@ func New(origArgs []string) *Options {
 	fs.StringVar(&opts.ListenAddress, "listen-address", "", "The listen address for the sentry server")
 	fs.IntVar(&opts.HealthzPort, "healthz-port", 8080, "The port for the healthz server to listen on")
 	fs.StringVar(&opts.HealthzListenAddress, "healthz-listen-address", "", "The listening address for the healthz server")
+	fs.StringVar(&opts.Mode, "mode", string(modes.KubernetesMode), "Runtime mode for Dapr Sentry")
```

Comment:
Default mode should be selfhosted, and set to kubernetes in the helm chart. We should check mode is either self hosted or kube else error.

Reply:
right, the helm chart is the one that sets this, that makes sense

Reply:
@lrascao please add this check ^

Reply:
I’m also assuming some int tests will fail as they were relying on the old “IsKubernstesHosted” method of checking, and so now instead we need to supply this flag. Could be wrong tho

Reply:
> @lrascao please add this check ^

check is being done in `opts.Validate`, unless you had some other in mind?

Reply:
Missed it, was expecting it in the options.New func